src/components/RecipeContainer.js

```diff
@@ -6,12 +6,12 @@ import sampleRecipes from '../sample-recipe';
 const RecipeContainer = (props) =>{
     const recipeId = props.match.params.recipeId;
 
-    const [recipe, singleRecipe] = useState({});
+    const [recipe, setSingleRecipe] = useState({});
```

Comment:
I'm going to ask again, plz change this to `setRecipe`, setSingleRecipe is redundant with the singular `recipe`